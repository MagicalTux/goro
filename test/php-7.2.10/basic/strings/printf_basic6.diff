--
     int(6)
     
     -- Calling printf() with one argument --
008- 1.000000e+3
009- int(11)
008+ 1.00000000000000e+03
009+ int(20)
     
     -- Calling printf() with two arguments --
012- 1.000000E+3 2.000000e+3
013- int(23)
012+ 1.00000000000000E+03 2.00000000000000e+03
013+ int(41)
     
     -- Calling printf() with three arguments --
016- 1.000000e+3 2.000000E+3 3.000000e+3
017- int(35)
016+ 1.00000000000000e+03 2.00000000000000E+03 3.00000000000000e+03
017+ int(62)
     ===DONE===
