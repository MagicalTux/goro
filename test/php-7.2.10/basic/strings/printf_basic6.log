
---- EXPECTED OUTPUT
*** Testing printf() : basic functionality - using exponential format ***

-- Calling printf() with no arguments --
format
int(6)

-- Calling printf() with one argument --
1.000000e+3
int(11)

-- Calling printf() with two arguments --
1.000000E+3 2.000000e+3
int(23)

-- Calling printf() with three arguments --
1.000000e+3 2.000000E+3 3.000000e+3
int(35)
===DONE===
---- ACTUAL OUTPUT
*** Testing printf() : basic functionality - using exponential format ***

-- Calling printf() with no arguments --
format
int(6)

-- Calling printf() with one argument --
1.00000000000000e+03
int(20)

-- Calling printf() with two arguments --
1.00000000000000E+03 2.00000000000000e+03
int(41)

-- Calling printf() with three arguments --
1.00000000000000e+03 2.00000000000000E+03 3.00000000000000e+03
int(62)
===DONE===
---- FAILED
