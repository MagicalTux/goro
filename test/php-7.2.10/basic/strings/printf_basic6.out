*** Testing printf() : basic functionality - using exponential format ***

-- Calling printf() with no arguments --
format
int(6)

-- Calling printf() with one argument --
1.00000000000000e+03
int(20)

-- Calling printf() with two arguments --
1.00000000000000E+03 2.00000000000000e+03
int(41)

-- Calling printf() with three arguments --
1.00000000000000e+03 2.00000000000000E+03 3.00000000000000e+03
int(62)
===DONE===